snapshot/storage/bolt.go

```diff
@@ -129,12 +130,12 @@ func CreateActive(ctx context.Context, key, parent string, readonly bool) (a Act
 			}
 
 			if parentS.Kind != db.KindCommitted {
-				return errors.Wrap(snapshot.ErrSnapshotNotCommitted, "parent is not committed snapshot")
+				return errors.Wrap(errdefs.ErrFailedPrecondition, "parent is not committed snapshot")
```

Comment:
I think this is just invalid argument. There is no operation which could be run to satisfy this condition for the parent key. The parent key would need to be committed under a different name, making it a different argument. I am assuming we are basing the use of these errors off the descriptions in https://godoc.org/grpc.go4.org/codes#Code